### ISSUE TITLE
Fake docs up to date?

### ISSUE THREAD

**Issue by OnurGumus** [SteveGilham/altcover#93]
Hello,
I am trying to use alt cover with fake wrt 

https://stevegilham.github.io/altcover/Fake-and-Cake-integration

https://stevegilham.github.io/altcover/AltCover.Fake.DotNet.Testing.AltCover/

But my impression is this code is not up to date. Please help.

**Comment by SteveGilham** [comment]
Well spotted.  I missed those fragments in the v7.0 mass update to everything -- should be fixed now in [the latest check-in](https://github.com/SteveGilham/altcover/commit/ead6c70d2120c385595b1028f73de1335cee3592)

**Comment by OnurGumus** [comment]
The updated version still doesn't compile.

**Comment by OnurGumus** [comment]
Basically I have up to this point
```

open AltCoverFake.DotNet
Target.create "RunTests" (fun _ ->
    let ForceTrue = DotNet.CLIOptions.Force true 

    let p =
      { Primitive.PrepareOptions.Create() with
          CallContext = [| "[Fact]"; "0" |]
          AssemblyFilter = [| "xunit" |] }

    let prep = AltCover.PrepareOptions.Primitive p
    let c = Primitive.CollectOptions.Create()
    let collect = AltCover.CollectOptions.Primitive c

    let setBaseOptions (o: DotNet.Options) =
      { o with
          WorkingDirectory = Path.getFullName "./_DotnetTest"
          Verbosity = Some DotNet.Verbosity.Minimal }

    failwith ""
    DotNet.test
        (fun to' -> { to'.WithCommon(setBaseOptions).WithAltCoverOptions prep collect ForceTrue })
         "dotnettest.fsproj"
)
```
The above part works but the fun to line doesn't. I don't understand why there are curly braces around to and there doesn't seem any extension methods avaialble withAltCoverOptions

**Comment by SteveGilham** [comment]
The curly brackets are a vestige of extra clutter I removed (but not thoroughly enough, it seems); the extension method is [here](https://stevegilham.github.io/altcover/AltCover.Fake.DotNet.Testing.AltCover/Fake-fsapidoc) or [here](https://stevegilham.github.io/altcover/AltCover.Fake/Fake-fsapidoc) depending on context (in-process or out of process).

I've fixed the obvious glitches, and also added links to the build script from which the examples are drawn.  What have I missed this time?
